### PR TITLE
feat(li): add correlation insight

### PR DESCRIPTION
**Pull request by Esorakouki**
### PR includes
- Add `correlation` insight
image
- add dev dep @rollup/plugin-json
image

TODO:  Subject description for 1M * 1M
image

### CODE REVIEW

File: packages/lite-insight/src/narrative/strategy/correlation.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * @example: There is a correlation between GDP and life_expect.
+ */
+export function correlationStrategy(variableMap: CorrelationInfo) {
+  const { measures } = variableMap;
+  const phrases = new PhrasesBuilder();
+  phrases.add('There is a correlation between');
+  phrases.add(measures[0], 'metric_name');
+  phrases.add('and');
+  phrases.add(measures[1], 'metric_name');
+  phrases.addSymbol('punctuation_stop'); // .
```
Comment (pddpd) [inline packages/lite-insight/src/narrative/strategy/correlation.ts:15]: What is this comment

File: packages/lite-insight/src/pipeline/extract.ts [hunk 2]
```diff
@@ -129,6 +129,47 @@ export const extractInsightsFor1M1DCombination = (
...
+  data: Datum[],
+  dimensions: string[],
+  measures: Measure[],
+  subspace: Subspace,
+  referenceInfo: ReferenceInfo,
+  options: InsightOptions
+): InsightInfo<PatternInfo>[] => {
+  const { fieldPropsMap } = referenceInfo;
+
+  const insights = [];
```
Comment (pddpd) [inline packages/lite-insight/src/pipeline/extract.ts:142]: Can you give it a type?

File: rollup.config.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import typescript from '@rollup/plugin-typescript';
+import json from '@rollup/plugin-json';
```
Comment (pddpd) [inline rollup.config.js:4]: import it for?

File: packages/lite-insight/src/pipeline/extract.ts (hunk 2 above)
Comment (Esorakouki) [reply packages/lite-insight/src/pipeline/extract.ts]: done

File: packages/lite-insight/src/narrative/strategy/correlation.ts (hunk 1 above)
Comment (Esorakouki) [reply packages/lite-insight/src/narrative/strategy/correlation.ts]: deleted

File: rollup.config.js (hunk 3 above)
Comment (Esorakouki) [reply rollup.config.js:4]: As is shown in the PR desc, an error occurred in build process.

File: packages/lite-insight/src/insights/checkers.ts [hunk 4]
```diff
@@ -62,10 +62,25 @@ export const lowVarianceChecker: ExtractorChecker = (data, subjectInfo, fieldPro
   return fieldPropsMap[measures[0].field].distinct !== 1;
 };
 
+export const correlationChecker: ExtractorChecker = (data, subjectInfo, fieldPropsMap) => {
```
[the remark this answers is not in the document]
Comment (Esorakouki) [reply packages/lite-insight/src/insights/checkers.ts:65]: there is a Func type `ExtractorChecker `

File: rollup.config.js (hunk 3 above)
Comment (pddpd) [reply rollup.config.js:4]: Okay, `@stdlib/stats` needs this plugin.

**Review by pddpd — APPROVED** [review]

**Pull request merged**